### ISSUE TITLE
ONNX/Caffe2 model conversion issue

### ISSUE THREAD

**Issue by Ashutosh1995** [qfgaohao/pytorch-ssd#136]
Hi,

Thanks for the repo containing all the models!

While converting pretrained mobilnet v1 SSD model to caffe2 and onnx, there is no predictor layer attached at the end because of which we only supply outputs as scores and boxes and miss out labels in the onnx conversion file.

How should I get a complete model in onnx which also has a predictor head and gives class names at the output ?

**Comment by shashwat623** [comment]
> Hi,
> 
> Thanks for the repo containing all the models!
> 
> While converting pretrained mobilnet v1 SSD model to caffe2 and onnx, there is no predictor layer attached at the end because of which we only supply outputs as scores and boxes and miss out labels in the onnx conversion file.
> 
> How should I get a complete model in onnx which also has a predictor head and gives class names at the output ?

@Ashutosh1995 I'm facing the same issue. Any updates ?
